src/pages/journey/index.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`Line 16:` use named exports instead of exporting index rename to MyJourney

Reply:
The component should changed, and capitalized